website/docs/r/data_factory_trigger_schedule.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_factory_trigger_schedule"
+sidebar_current: "docs-azurerm-resource-data-factory-trigger-schedule"
+description: |-
+  Manages a Trigger Schedule inside a Azure Data Factory.
+---
+
+# azurerm_data_factory_trigger_schedule
+
+Manages a Trigger Schedule inside a Azure Data Factory.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example"
+  location = "northeurope"
+}
+
+resource "azurerm_data_factory" "example" {
+  name                = "example"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+}
+
+resource "azurerm_data_factory_trigger_schedule" "test" {
+  name                = "acctestdf%d"
+  data_factory_name   = "${azurerm_data_factory.test.name}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+  interval    = 5
+  frequency   = "Day"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Data Factory Schedule Trigger. Changing this forces a new resource to be created. Must be globally unique. See the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/data-factory/naming-rules) for all restrictions.
```

Comment:
If the API has a `CheckName1 function we should probably make use of it to ensure the name is unique

Reply:
Unfortunately, I didn't find one